SIL.Windows.Forms/FileSystem/ConfirmFileOverwriteDlg.cs

```diff
@@ -0,0 +1,82 @@
+// ---------------------------------------------------------------------------------------------
+#region // Copyright (c) 2013, SIL International.
```

Comment:
2023

Reply:
It was originally written in 2013. Just moved it from Transcelerator. But the namespace did change (and it does look weird in the history), so I've updated it.